### PR TITLE
Add URL generation functions to TaskRouter JWTs

### PR DESCRIPTION
**Pull request by jingming**

### CODE REVIEW

File: lib/jwt/taskrouter/util.js [hunk 1]
```diff
@@ -65,7 +66,99 @@ function defaultEventBridgePolicies(accountSid, channelId) {
   ];
 }
 
+/**
+ * Generate TaskRouter workspace url
+ *
+ * @param {string} [workspaceSid] workspace sid or '**' for all workspaces
```
Comment (efossier) [inline lib/jwt/taskrouter/util.js]: maybe just default it to this if param is undefined so that people can just call `workspacesUrl()`?

**Comment by efossier** [comment]
Seems reasonable

File: lib/jwt/taskrouter/util.js (hunk 1 above)
Comment (jingming) [inline lib/jwt/taskrouter/util.js]: you should be able to do that here. Is this what you are referring to? https://github.com/twilio/twilio-node/pull/246/files#diff-cce12a1e86ded2760558610cfe5ef0bcR16

Comment (efossier) [inline lib/jwt/taskrouter/util.js]: Ah it looks like that will produce different behavior though, I guess thats what we want though. carry on.

**Pull request merged**